### ISSUE TITLE
Cannot disable circleMarker draw UI

### ISSUE THREAD

**Issue by benjimin** [jupyter-widgets/ipyleaflet#106]
By default, `ipyleaflet.DrawControl` facilitates drawing polylines, polygons, and circle markers. I want to be able to restrict this to polygons only. Is there any way of removing the circle marker control from the toolbar?
```python
m = ipyleaflet.Map(center=(-27,134), zoom=4)
c = ipyleaflet.DrawControl(circle={}, circleMarker={}, circlemarker={}, 
                           CircleMarker={}, polyline={}, marker={}, polygon={}) 
m.add_control(c)
m
```
In ipyleaflet 0.6.0 (also tried 0.4.0) this only disables the polygon and polyline.
![image](https://user-images.githubusercontent.com/12852539/35833997-c3232032-0b27-11e8-941f-a2a96280cb5d.png)

**Comment by benjimin** [comment]
Also is there any way _from python_ to obtain a list of currently drawn polygons (not just `.last_draw`), and to remove a drawn polygon from the map?

**Comment by martinRenou** [comment]
This issue should be fixed by #194, if it gets merged, you'd only need to write 
```python
m = ipyleaflet.Map(center=(-27,134), zoom=4)
c = ipyleaflet.DrawControl(circlemarker={}, polyline={}, polygon={}) 
m.add_control(c)
m
```
in order to get what you expect
